pkg/controller.v1alpha3/experiment/util/status_util.go

```diff
@@ -50,26 +50,28 @@ func UpdateExperimentStatus(collector *ExperimentsCollector, instance *experimen
 
 func updateTrialsSummary(instance *experimentsv1alpha3.Experiment, trials *trialsv1alpha3.TrialList) bool {
 
-	var totalTrials, trialsPending, trialsRunning, trialsSucceeded, trialsFailed, trialsKilled int32
 	var bestTrialValue float64
+	sts := &instance.Status
+	sts.Trials = 0
+	sts.RunningTrialList, sts.PendingTrialList, sts.FailedTrialList, sts.SucceededTrialList, sts.KilledTrialList = nil, nil, nil, nil, nil
 	bestTrialIndex := -1
 	isObjectiveGoalReached := false
 	objectiveValueGoal := *instance.Spec.Objective.Goal
 	objectiveType := instance.Spec.Objective.Type
 	objectiveMetricName := instance.Spec.Objective.ObjectiveMetricName
 
 	for index, trial := range trials.Items {
-		totalTrials++
+		sts.Trials++
 		if trial.IsKilled() {
-			trialsKilled++
+			sts.KilledTrialList = append(sts.KilledTrialList, trial.Name)
 		} else if trial.IsFailed() {
-			trialsFailed++
+			sts.KilledTrialList = append(sts.KilledTrialList, trial.Name)
```

Comment:
Should it be `sts.FailedTrialList` instead of `sts.KilledTrialList` ?

Reply:
@andreyvelich Yes, I've already fixed it.